pkg/disk/partition_table_test.go

```diff
@@ -2531,6 +2531,100 @@ func TestNewCustomPartitionTable(t *testing.T) {
 				},
 			},
 		},
+		"plain-ukiboot": {
+			customizations: &blueprint.DiskCustomization{
+				Type: "gpt",
+				Partitions: []blueprint.PartitionCustomization{
+					{
+						Type:    "plain",
+						MinSize: 64 * datasizes.MiB,
+						FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+							Label: "ukiboot_a",
+						},
+					},
+					{
+						Type:    "plain",
+						MinSize: 64 * datasizes.MiB,
+						FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+							Label: "ukiboot_b",
+						},
+					},
+					{
+						Type:    "plain",
+						MinSize: 1 * datasizes.MiB,
+						FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+							Label: "ukibootctl",
+						},
+					},
+				},
+			},
+			options: &disk.CustomPartitionTableOptions{
+				BootMode:           platform.BOOT_UEFI,
+				PartitionTableType: disk.PT_GPT,
+				DefaultFSType:      disk.FS_XFS,
+				Architecture:       arch.ARCH_X86_64,
+			},
+			expected: &disk.PartitionTable{
+				Type: disk.PT_GPT,
+				Size: 331 * datasizes.MiB,
```

Comment:
(super nitpick) in some places we do `1+200+64+64+3` to match the size with the partitions in the slice below, same with "Start", it makes things slighly simpler if one has to tweak the test and change sizes but a) its rare b) we are not very consistent about it. So feel free to ignore :)

Reply:
Yes, that makes sense too. I'll change it.